modules/commands/list.json

```
@@ -296,6 +306,14 @@
     "description": "Learn what each of the config options are for.",
     "wiki": true
   },
+  {
+    "name": "extracontexts",
+    "aliases": "extracontext",
+    "title": "Extra Contexts",
+    "url": "https://github.com/lucko/LuckPerms/wiki/Context#ExtraContexts",

```

We can just link to https://github.com/LuckPerms/ExtraContexts
```suggestion
    "url": "https://github.com/LuckPerms/ExtraContexts",
```